README.md

```diff
@@ -182,10 +182,11 @@ provided keymappings file. Follow the lua guide available
 
 # Install your own plugins
 
-The steps for configuring your own plugin are: 1. Add the plugin to
-plugins.lua 2. If the plugin requires configuration. Create a
-configuration file for it 3. If you created a configuration, require the
-file in init.lua 4. Use Packer to download and install the plugin
+The steps for configuring your own plugin are: 
+1. Add the plugin to `plugins.lua`
+2. If the plugin requires configuration, create a configuration file for it
+3. If you created a configuration, require the file in `init.lua`
+4. Use Packer to download and install the plugin
```

Comment:
Let Markdown render the actual number automatically, this way it is easier to mantain.
```suggestion
1. Add the plugin to `plugins.lua`
1. If the plugin requires configuration, create a configuration file for it
1. If you created a configuration, require the file in `init.lua`
1. Use Packer to download and install the plugin
```